R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -74,32 +78,34 @@ importBIOM <- function(file, ...) {
 
 #' @rdname makeTreeSEFromBiom
 #'
-#' @param obj object of type \code{\link[biomformat:read_biom]{biom}}
+#' @param x object of type \code{\link[biomformat:read_biom]{biom}}
 #'
 #' @export
 #' @importFrom S4Vectors make_zero_col_DFrame DataFrame
 #' @importFrom dplyr %>% bind_rows
 makeTreeSEFromBiom <- function(
-        obj, removeTaxaPrefixes = FALSE, rankFromPrefix = FALSE,
+        x, prefix.rm = removeTaxaPrefixes, 
+        removeTaxaPrefixes = FALSE, rank.from.prefix = rankFromPrefix, 
+        rankFromPrefix = FALSE,
         remove.artifacts = FALSE, ...){
     # input check
     .require_package("biomformat")
-    if(!is(obj,"biom")){
-        stop("'obj' must be a 'biom' object", call. = FALSE)
+    if(!is(x,"biom")){
+        stop("'x' must be a 'biom' object", call. = FALSE)
     }
-    if( !.is_a_bool(removeTaxaPrefixes) ){
-        stop("'removeTaxaPrefixes' must be TRUE or FALSE.", call. = FALSE)
+    if( !.is_a_bool(prefix.rm) ){
+        stop("'prefix.rm' must be TRUE or FALSE.", call. = FALSE)
     }
-    if( !.is_a_bool(rankFromPrefix) ){
-        stop("'rankFromPrefix' must be TRUE or FALSE.", call. = FALSE)
+    if( !.is_a_bool(rank.from.prefix) ){
+        stop("'rank.from.prefix' must be TRUE or FALSE.", call. = FALSE)
     }
     if( !.is_a_bool(remove.artifacts) ){
         stop("'remove.artifacts' must be TRUE or FALSE.", call. = FALSE)
```

Comment:
`rm.artifacts`?

Reply:
To follow the naming convention `na.rm`, `prefix.rm`... this should be `artifact.rm`

Reply:
@Daenarys8 artifact.rm

Reply:
ACK.